tests/test_kms.py

```diff
@@ -39,4 +39,36 @@ def test_key_rotation(self):
             session_factory=session_factory)
 
         resources = p.run()
-        self.assertEqual(len(resources), 7)
+        self.assertEqual(len(resources), 2)
+
+    def test_kms(self):
```

Comment:
I think my motivation was to ensure that kms key-alias augment ensures mapping with a valid key. (https://github.com/capitalone/cloud-custodian/blob/master/c7n/resources/kms.py#L48) 

You're right this isn't really adding value as a test. I'll review and remove this and any other test that doesn't make sense.